src/content/LandingPage/LandingPage.js

```diff
@@ -1,7 +1,98 @@
 import React from 'react';
-
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Tabs,
+  Tab,
+} from 'carbon-components-react';
+const props = {
+  tabs: {
+    selected: 0,
+    triggerHref: '#',
+    role: 'navigation',
+  },
+  tab: {
+    href: '#',
+    role: 'presentation',
+    tabIndex: 0,
+  },
+};
 const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+  return (
+    <div className="bx--grid bx--grid--full-width landing-page">
+      <div className="bx--row landing-page__banner">
+        <Breadcrumb noTrailingSlash aria-label="Page navigation">
+          <BreadcrumbItem>
+            <a href="/">Getting started</a>
+          </BreadcrumbItem>
+        </Breadcrumb>
+        <h1 className="landing-page__heading">
+          Design &amp; build with Carbon
+        </h1>
+      </div>
```

Comment:
Your Getting Started and Design & Build with Carbon heading are all on the same line.

Reply:
How do I recover step 2, to resolve and re-submit? I did have problems before with recovering my step 2.

Reply:
You need to check out the branch. 

See
https://www.atlassian.com/git/tutorials/using-branches/git-checkout

Reply:
I know how to do that (checkout), but for some reason I have 1 uncommited change to upstream, and do not know how to clear it out. 
 I have tried 'git reset --hard '

Not sure how to not have it attempt to push upstream.

![2019-10-14_11-33-13](https://user-images.githubusercontent.com/42783294/66764042-08f05d80-ee77-11e9-9483-773561c26bad.png)

Reply:
You cannot push to upstream as that is the carbon-design-system repo. You must push to origin. 

See https://www.carbondesignsystem.com/tutorial/react/step-2#git-commit-and-push

Reply:
See forced push...but not sure that worked. I seem to really have messed up my git repo with force option.